keyboards/iris/keymaps/yanfali/config.h

```diff
@@ -27,8 +27,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 /* Select hand configuration */
 
-#define MASTER_LEFT
-// #define MASTER_RIGHT
+//#define MASTER_LEFT
```

Comment:
This change should be reset by reverting to the original version.

`git checkout master -- keyboards/iris/keymaps/yanfali/config.h`

Reply:
This change still needs to be undone.

Reply:
Oops, sorry @yanfali ;)